docs/minishift_docker-env.md

```diff
@@ -23,13 +23,15 @@ minishift docker-env
 
 ```
       --alsologtostderr                  log to standard error as well as files
+      --http-proxy string                HTTP proxy for virtual machine (ex. http://<username>:<password>@foo.com:<proxy_port>)
+      --https-proxy string               HTTPS proxy for virtual machine (ex. https://<username>:<password>@foo.com:<proxy_port>)
       --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
       --log_dir string                   If non-empty, write log files in this directory (default "")
       --logtostderr                      log to standard error instead of files
-      --password string                  Password for the virtual machine.
+      --password string                  Password for the virtual machine registration.
```

Comment:
hmm, here the global --no-proxy does not get added. I guess that's because the command itself has a no-proxy option. Do we have a conflict there?